src/core/components/SideMenuSection/index.js

```diff
@@ -5,18 +5,28 @@ import styled from 'styled-components';
 // TODO - confirm colour with design
```

Comment:
is this still a TODO? also, maybe we could keep the colors as constants somewhere? it will be easier to change